Microsoft.Azure.Cosmos.Encryption/src/EncryptionContainer.cs

```diff
@@ -825,5 +704,378 @@ internal async Task InitEncryptionContainerCacheIfNotInitAsync(CancellationToken
         {
             throw new NotImplementedException();
         }
+
+        public override Task<ResponseMessage> ReadManyItemsStreamAsync(
+            IReadOnlyList<(string id, PartitionKey partitionKey)> items,
+            ReadManyRequestOptions readManyRequestOptions = null,
+            CancellationToken cancellationToken = default)
+        {
+            throw new NotImplementedException();
+        }
+
+        public override Task<FeedResponse<T>> ReadManyItemsAsync<T>(
+            IReadOnlyList<(string id, PartitionKey partitionKey)> items,
+            ReadManyRequestOptions readManyRequestOptions = null,
+            CancellationToken cancellationToken = default)
+        {
+            throw new NotImplementedException();
+        }
+
+        public async Task<EncryptionSettings> GetOrUpdateEncryptionSettingsFromCacheAsync(
+            CancellationToken cancellationToken,
+            EncryptionSettings obsoleteEncryptionSettings = null)
+        {
+            return await this.encryptionSettingsByContainerName.GetAsync(
+                this.Id,
+                obsoleteValue: obsoleteEncryptionSettings,
+                singleValueInitFunc: async () => await EncryptionSettings.CreateAsync(this),
```

Comment:
You shouldn't need the wrapper delegate.
```suggestion
                singleValueInitFunc: this.EncryptionSettings.CreateAsync(this),
```

Reply:
Fixed.